install_config/configuring_vsphere.adoc

```diff
@@ -157,6 +157,27 @@ determined by using the command `uname -n`. In case of a mismatch, the native
 cloud integration for those providers will not work.
 ====
 
+== Configuring OpenShift Container Platform for vSphere using Ansible
```

Comment:
Add `[[vsphere-configuring-ansible]]` on line 159.

Reply:
Thank you for your review. I'm done.